CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ again.
      issue that you are planning to work on that bug or feature so that it can
      be assigned to you.
 
-  1. Follow the normal process of [forking][] the project, and setup a new
+  1. Follow the normal process of [forking][] the project, and setup() a new
```

Comment:
This change looks unintentional.

Reply:
Thanks for spotting, I have fixed this now.